tests/unit/App.spec.ts

```diff
@@ -801,12 +801,7 @@ describe('App as a Name Request', () => {
   })
 
   it('fetches Tasks properly', () => {
-    expect(vm.$store.state.tasks.length).toBe(1)
-    expect(vm.$store.state.tasks[0].enabled).toBe(true)
-    expect(vm.$store.state.tasks[0].order).toBe(1)
-    expect(vm.$store.state.tasks[0].task.todo.nameRequest).not.toBeUndefined()
-    expect(vm.$store.state.tasks[0].task.todo.header.name).toBe('nameRequest')
-    expect(vm.$store.state.tasks[0].task.todo.header.status).toBe('NEW')
+    expect(vm.$store.state.tasks.length).toBe(0)
```

Comment:
You should leave the old code but return a task in the mock above (line 756).

Reply:
Great tests, thanks!

Reply:
This is restored and one more test to TodoList